### ISSUE TITLE
Can reproduce the results of 'Hopper-v2'

### ISSUE THREAD

**Issue by lknownothing** [ku2482/simple-rl.pytorch#1]
Thank you for sharing the clear code!

This code can work well with `Walker2d-v2, Ant-v2, HalfCheetah-v2`. but When I used it to reproduce `Hopper-v2`, After 1M training step, The reward is about ~2200 and very unstable.

I've checked the code several times and I believe it's correct, but I can't get the same results as the original paper on 'Hopper-v2'. I don't know why it can't work steadily on `Hopper-v2`.
